similar patch to use libm was alreadu accepted by upstream. Need to do also this one.
Sent upstream
https://bugzilla.mozilla.org/show_bug.cgi?id=1351309

--- firefox-52.0/js/src/jit-test/tests/ctypes/function-definition.js
+++ firefox-52.0/js/src/jit-test/tests/ctypes/function-definition.js
@@ -27,7 +27,7 @@
 
   let lib;
   try {
-    lib = ctypes.open(ctypes.libraryName("c"));
+    lib = ctypes.open(ctypes.libraryName("m"));
   } catch (e) {
   }
   if (!lib)
